cpp/marc_tools/single_marc_convert_keywords.cc

```diff
@@ -0,0 +1,149 @@
+/** \brief Utility for comparing keywords with gnd database.
+ *  \author Hjordis Lindeboom
+ *
+ *  \copyright 2021 Universitätsbibliothek Tübingen.  All rights reserved.
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#include <fstream>
+#include <iostream>
+#include <set>
+#include <stdexcept>
+#include <utility>
+#include <climits>
+#include <cstdio>
+#include <cstdlib>
+#include "FileUtil.h"
```

Comment:
You not using this anywhere.  Please remove it.  Also this PR is still set to Draft!